server/utils/statemanager/waiting_for_players_state.go

```diff
@@ -22,6 +22,10 @@ func (state waitingForPlayersState) startGame(groupName string, playerName strin
 	return nil
 }
 
-func (state waitingForPlayersState) addPrompt(prompt *models.Prompt) error {
-	return errors.New("addprompt not supported for waiting for players state")
+func (state waitingForPlayersState) submitDrawing(playerName string, encodedImage string) error {
```

Comment:
it seems odd that most classes don't implement most functions.

Reply:
Yeah, it's kind of overkill right now but I think that:
- AddPrompt will make a difference depending on which prompt-ey state we're in (fake prompts or real prompts)
- Formatting the state response for a given player will also vary a lot by state